addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -297,6 +297,9 @@ def _get_store_partner_fields(self, fields):
     # RTC (voice/video)
     # --------------------------------------------------------------------------
 
+    def _should_invite_members(self):
```

Comment:
It might be worth being a little bit more specific on the name, as it is closely related to the "join call" flow.